docs-ref-conceptual/azure-cli-vm-tutorial.yml

```diff
@@ -183,7 +183,7 @@ items:
         the shell `read` command to load results into multiple variables. Since two values on separate lines are displayed, the `read` command
         delimiter must be set to the empty string rather than the default of non-newline whitespace.
 
-        ```bash
+        ```azurecli-interactive
```

Comment:
The contributor guide made it sound like they were discouraging the use of azurecli-interactive (its use is under discussion).

Reply:
I used it here because it was the second or third of a series that already used azurecli-interactive. I thought it would stick out if I didn't.